src/parquet/util/bit-stream-utils.inline.h

```diff
@@ -75,7 +75,7 @@ inline bool BitWriter::PutAligned(T val, int num_bytes) {
   return true;
 }
 
-inline bool BitWriter::PutVlqInt(int32_t v) {
+inline bool BitWriter::PutVlqInt(uint32_t v) {
```

Comment:
I think this just does the right thing. The implicit cast from signed to unsigned is correct.